source/api/image/3.0/index.md

```diff
@@ -846,6 +855,7 @@ The following shows an image information response including all of the required
     }
   ],
   "rights": "http://rightsstatements.org/vocab/InC-EDU/1.0/",
+  "preferredFormats": [ "png", "gif"],
```

Comment:
Very very minor point - we could use webp rather than (or as well as) gif in the example... [ "webp", "png" ]